modmesh/pilot/_base_app.py

```diff
@@ -671,6 +722,40 @@ def init_ui(self):
         self.setLayout(layout)
 
 
+class FigureConfigDialog(QDialog):
+    """
+    PlotConfigDialog class for managing plot configuration.
+
+    This class inherits from the QDialog class and provides a GUI for managing
+    plot configuration. It will pop up when clicking "Option" button. It
+    includes options for selecting the layout and setting data line config.
+    """
+    def __init__(self, app, parent=None):
+        super().__init__(parent)
+        self.app = app
+        self.init_ui()
+
+    def init_ui(self):
+        self.setWindowTitle("Figure Options")
+        layout = QVBoxLayout()
+
+        for [name, *_] in self.app.config_option:
```

Comment:
Do not use a list (`[]`) between `for` and `in`.  The following works the same:

```python
for name, *_ in self.app.config_option:
```

See

```python
>>> data = [[1, 2], [3, 4, 5]]
>>> for i, *_ in data:
...     print(i)
...
1
3
```